cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -80,6 +80,9 @@ message Circuit {
   // This field is used to reduce size of circuits that contain many
   // repeated moments.
   repeated int32 moment_indices = 3;
+
+  optional int32 token_constant_index = 4;
```

Comment:
Passing by out of curiosity :) 

Could you add more comments on the new fields about how the index / indices are used in representing and constructing the Circuit?

Reply:
You can probably just copy the comments from the identical fields on the Operation message.